src/modules/formBuilder/components/formTree/FormTreeLabel.tsx

```diff
@@ -91,6 +112,47 @@ const FormTreeLabel: React.FC<FormTreeLabelProps> = ({
         height: '48px',
       }}
     >
+      {!!itemDependents && (
+        <ConfirmationDialog
+          open={!!itemDependents}
+          title='Cannot delete item'
+          onConfirm={() => setItemDependents(undefined)}
+          loading={false}
+          hideCancelButton={true}
+        >
+          "{displayLabelForItem(item)}" cannot be deleted because it is
+          referenced elsewhere.
+          <Alert
+            color='error'
+            icon={false}
+            sx={{ my: 2, '& .MuiAlert-message': { width: '100%' } }}
+          >
+            {Object.entries(itemDependents).map(([key, val]) =>
+              val?.length > 0 ? (
+                <Box key={key}>
+                  {dependentLabelMap[key]}:
+                  <List sx={{ listStyleType: 'disc', py: 0 }}>
```

Comment:
This would probably be a good opportunity to add a `CommonUnorderedList` component, that can be used like so:
```
<CommonUnorderedList>
  <li> item 1 </li>
  <li> item 2 </li>
</CommonUnorderedList>
```


That matches the pattern we use for `CommonUnstyledList` and `CommonOrderedList` components. They make use of simple ol/ul tags instead of MUI which is a bit heavy-weight for this use case. This will also help ensure that future bulleted lists we place in the app will have the same spacing. (Assuming the `CommonUnorderedList` component is discoverable).

See usage here: https://github.com/greenriver/hmis-frontend/blob/5a274046ef33057613c29a37aa16053790bf8197/src/modules/assessments/components/household/HouseholdSummaryIntakeHelpCard.tsx#L44-L51